### DISCUSSION TITLE
Is there a way to add custom data to a listing submitted through the Etsy Api?

### DISCUSSION THREAD

**Discussion by tahirkahnicloud** [etsy/open-api#988] [Q&A] [answered]
Hey fellow devs,
Is there a way to add custom data to a listing submitted through the listing API? For instance one may want to add a custom warehouse SKU to corelate a listing created on Etsy with the inventory in the warehouse system. Is there something that exists in V3 for this? Also, I did not find a way to get category suggestions. Not sure but I think this existed in v2 and I do not see it in V3. Other than creating custom mapping algo in code, is there a simpler way of getting cat suggestions?

Thanks a mil!

**Comment by Gareth064** [comment, marked as answer]
Hey @tahirkahnicloud.

You can assign your own SKUs using the inventory endpoints.

As for suggested categories, can you raise that as a separate discussion? That will keep these two questions separate and easier for others to find if they search something similar.

**Comment by tahirkahnicloud** [comment]
I totally overlooked this! Thanks and I will create another thread for the category suggestion
